onset_maproom/cess.py

```diff
@@ -20,38 +20,36 @@
 rr_mrg = calc.read_zarr_data("/data/aaron/ethiopia-rain-rechunked")
 
 #subset of data for testing
-rr_mrgSub = rr_mrg.sel(T=slice('1981-01-01','1983-12-31'))
-daily_rain = rr_mrgSub.isel(X=200,Y=160)
+rr_mrgSub = rr_mrg.sel(T=slice('1983-01-01','1986-12-31'))
+daily_rain = rr_mrgSub.sel(X="37.75",Y="9", method="nearest", tolerance=0.04)
 
 
 #onset date calc used in seasonal cess date func
 #first date after START_DAY(1), START_MONTH(9) in SEARCH_DAYS(90) 
 #days when the soil water balance falls below DRY_THRESH(5)mm 
 #for a period of DRY_SPELL_LENGTH(3) days
-def cess_date(daily_rain, dry_thresh, min_dry_days, et, taw, sminit, time_coord="T"):
-    water_balance = calc.water_balance(daily_rain, et,taw,sminit,"T")
+def cess_date(water_balance, dry_thresh, min_dry_days, et, taw, sminit, time_coord="T"):
```

Comment:
let's call the input soil_moisture rather than water_balance... there can be (and will be) many other variables in the water_balance dataset.
Also et, taw, sminit are no more arguments on cess_date since they are arguments of water_balance